services/web/server/tests/integration/test_exporter.py

```diff
@@ -287,7 +297,8 @@ async def import_study_from_file(client, file_path: Path) -> str:
 
 
 @pytest.mark.parametrize("export_version", get_exported_projects())
-async def test_import_export_import(
+async def test_import_export_import_duplicate(
+    loop,
```

Comment:
Should you check that s3 data was also duplicated?

Reply:
You are correct, this should be checked as well. I will compare that all imported files are the same (after being downloaded)